source/material_model/grain_size.cc

```diff
@@ -894,22 +889,17 @@ namespace aspect
                                                    this->get_adiabatic_conditions().temperature(in.position[i])
                                                    :
                                                    in.temperature[i];
-              const double adiabatic_pressure = this->get_adiabatic_conditions().is_initialized()
-                                                ?
-                                                this->get_adiabatic_conditions().pressure(in.position[i])
-                                                :
-                                                in.pressure[i];
 
               const double diff_viscosity = diffusion_viscosity(in.temperature[i],
                                                                 adiabatic_temperature,
-                                                                adiabatic_pressure,
+                                                                pressure,
```

Comment:
Should this (and the same place below) be `in.pressure[i]`?

Reply:
No it shouldn't. It's not in.pressure, it's the adiabatic pressure (to avoid pressure oscillations, as a comment somewhere way above says). I could rename the variable `pressure` to `adiabatic_pressure` if you think that would be better?

Reply:
Yes. I was just going by the use of `in.pressure[i]` previously.